packages/processor/processor.js

```diff
@@ -508,9 +508,7 @@ class Processor {
         this._addSelector(name, selector);
 
         refs.forEach(({ name : depSelector }) => {
-            const depSelectorId = this._addSelector(dep, depSelector);
-
-            graph.addDependency(selectorId, depSelectorId);
```

Comment:
It works thanks to that piece of code https://github.com/tivac/modular-css/blob/e026d812b17526a8f7e756205f5ed7424563dfee/packages/processor/plugins/composition.js#L77-L79